core/src/main/scala/com/microsoft/azure/synapse/ml/logging/Usage/TokenUtils.scala

```diff
@@ -0,0 +1,111 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
```

Comment:
im not sure why everything is under a package Usage. if you need to make a subpackage make it all lowercase and give it a reasonable name like fabric

Reply:
This contains code for reporting usage data. Hence, also the name usage.